.github/scripts/jirabot.py

```diff
@@ -17,6 +17,12 @@ def sanitize_input(input_str: str, input_type: str) -> str:
     if input_type.lower() == 'email':
         # Return the email address only, returns '' if not valid or found
         return parseaddr(input_str)[1]
+    elif input_type.lower() == 'text':
+        # Remove potentially dangerous characters
+        import re
+        # Allow alphanumeric, spaces, hyphens, underscores, and basic punctuation
+        sanitized = re.sub(r'[^\w\s\-._@#()[\]{},;:!?+=*/\\&%$]', '', input_str)
```

Comment:
I believe removing hyphens will break the subsequent Jira issue lookup. The remaining characters should be fine I think.